composer.json

```diff
@@ -78,7 +82,13 @@
     "post-create-project-cmd": [
       "@cleanup",
       "WordPressProject\\composer\\ScriptHandler::createRequiredFiles"
+    ],
+    "code-sniff": [
+      "./vendor/bin/phpcs --config-set installed_paths ./vendor/wp-coding-standards/wpcs",
+      "./vendor/bin/phpcs ./web/wp-content"
+    ],
+    "unit-test": [
+      "./vendor/bin/phpunit tests/unit/*"
```

Comment:
Only running the tests if there are files to be tested sounds reasonable.

My only objection would be of the reasons for the `.gitkeep` files is so that folks new to a lean repository can see that `wp-content/themes` and `wp-content/plugins` still exist and know where to add their custom code.